erela.cli.ts

```diff
@@ -164,17 +168,18 @@ const createFile = async (
   for (const folder of dir.split('/')) {
     path = join(path, folder)
     if (!(await existsAsync(path))) {
-      await mkdirAsync(path)
+      await mkdirAsync(`./${path}`)
     }
   }
-  await writeFileAsync(
-    join(__dirname, `${path}/${name.toLowerCase()}.${choice.toLowerCase()}.ts`),
-    content
-  )
-  log(
-    `plz look at ${path}/${name.toLowerCase()}.${choice.toLowerCase()}.ts`
-      .yellow.bold
+  const { cwd } = process
```

Comment:
this to get the current path that the cli has been run from 

English make sense?